dissect/target/plugins/browsers/chromium.py

```diff
@@ -89,6 +93,61 @@ def history(self, browser_name: str = None) -> Iterator[Record]:
             except SQLError as e:
                 self.target.log.warning("Error processing history file: %s", db_file, exc_info=e)
 
+    def downloads(self, browser_name: str = None) -> Iterator[BROWSER_DOWNLOAD_RECORD]:
+        """Return browser download records from supported Chromium-based browsers.
+
+        Args:
+            browser_name: The name of the browser as a string.
+        Yields:
+            Records with the following fields:
+                hostname (string): The target hostname.
+                domain (string): The target domain.
+                ts_start (datetime): Download start timestamp.
+                ts_ed (datetime): Download end timestamp.
+                browser (string): The browser from which the records are generated from.
+                id (string): Record ID.
+                path (string): Download path.
+                url (uri): Download URL.
+                size (varint): Download file size.
+                state (varint): Download state number.
+                source: (path): The source file of the history record.
+        Raises:
+            SQLError: If the file with the donwload records could not be processed.
+        """
+        for user, db_file, db in self._iter_db("History"):
+            try:
+                download_chains: dict = {}
```

Comment:
```suggestion
                download_chains = defaultdict(list)
```
And add a `from collections import defaultdict` import.